### ISSUE TITLE
"Use spaces, not tabs." Column Index

### ISSUE THREAD

**Issue by ionpot** [douglascrockford/JSLint#182]
```js
(function (i) {
	console.log(i);
}(0));
```
The `console.log` line is indented by one tab character. When I ran JSLint on it, I noticed the "Use spaces, not tabs." warning object had a column index of 1 instead of zero. Is this intentional?

**Comment by karimsa** [comment]
JSLint requires you to use spaces instead of tabs in indents. It is best practice to use 4 space characters as opposed to 1 tab character.

If you wish to disable this complaint, use the 'white' directive (i.e. `/*jslint white:true */`).

For more directives, see the help page: http://www.jslint.com/help.html

**Comment by ionpot** [comment]
Personally, I don't use tabs, but that's not the issue. The warning itself
is pointing to the character "c" (column 1) instead of the tab character
(column 0).

**Comment by mk-pmb** [comment]
The full message is probably this:
```text
 #1 Use spaces, not tabs.
    console.log(i); // Line 3, Pos 1
```
It has "Pos 1" because the tab character is the first character in that line.

**Comment by ionpot** [comment]
I was a bit vague describing the issue actually, apologies.

Thing is, I'm using the _jslint.js_ file as a Node.js module. I feed it JS source code and it returns an object for each warning.

One of the properties this warning object has is the _column_ property, which holds a number that points to the character in question.

It appears to be zero-indexed because I get 0 from it when the warning is at the first character. 

When it's a "Use spaces, not tabs." object, however, _column_ is set to 1, which is the character after the tab. Hence, I thought there might be an off-by-one error.

**Comment by douglascrockford** [comment]
It is the location of first token after the bad whitespace.
